src/index.html

```diff
@@ -17,11 +20,378 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/index.css"
     />
   </head>
 
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__body">
+    <header class="header">
+      <a
+        class="header__logo-link"
+        href="#"
+      >
+        <img
+          class="header__logo-img"
+          src="./images/logo.png"
+          alt="logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link nav__link--active is-active"
```

Comment:
The class attribute for this navigation link contains both 'nav__link--active' and 'is-active'. Ensure that both classes are necessary and not redundant. If they serve the same purpose, consider using only one.